### PR TITLE
[openscap]: fix not install oscap util

### PR DESCRIPTION
**Pull request by jiayuehua**
- [X] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [X] SHA512s are updated for each updated download
- [ ] ~The "supports" clause reflects platforms that may be fixed by this new version~
- [ ] ~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~
- [ ] ~Any patches that are no longer applied are deleted from the port's directory.~
- [X] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [X] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Review by Adela0814 — DISMISSED** [review]

File: ports/openscap/fix-utils.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/utils/CMakeLists.txt b/utils/CMakeLists.txt
+index f500e1d..641ab3a 100644
+--- a/utils/CMakeLists.txt
++++ b/utils/CMakeLists.txt
+@@ -24,7 +24,7 @@ if(ENABLE_OSCAP_UTIL)
+ 			target_link_libraries(oscap ${GETOPT_LIBRARY})
+ 		endif()
+ 		
+-		set(OSCAP_UTIL_DESTINATION ".")
++		set(OSCAP_UTIL_DESTINATION tools)
```
Comment (dg0yt) [inline ports/openscap/fix-utils.patch:10]: I would suggest to stay with the standard `vcpkg_cmake_configure` pattern: Install the tools to `bin` first, then move them with `vcpkg_copy_tools` (when tools are enabled). The latter function takes care of adding DLLs as needed.

Comment (jiayuehua) [reply ports/openscap/fix-utils.patch:10]: I will try to change tomorrow that following your suggestion .

Comment (dg0yt) [reply ports/openscap/fix-utils.patch:10]: Great.

I just checked the file lists from CI, and some other files caught my attention:
~~~
openscap:arm64-windows:/bin/autotailor
openscap:arm64-windows:/bin/oscap-run-sce-script
~~~
Both may be shell scripts and `vcpkg_copy_tools `won't handle them. They would need to be moved before `vcpkg_copy_tools` if you want to use the function's `AUTO_CLEAN` feature.

Comment (jiayuehua) [reply ports/openscap/fix-utils.patch:10]: you mean move autotailor and oscap-run-sce-script to tools/openscap/ ?

**Review by dg0yt — APPROVED** [review]
Thank you.

**Review by Adela0814 — DISMISSED** [review]

**Pull request merged**